### DISCUSSION TITLE
Required assets for widgets

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#175] [Developer Feedback]
According to [docs](https://docs.zepp.com/docs/reference/device-app-api/newAPI/ui/widget/SLIDE_SWITCH/) widgets 

- SLIDE_SWITCH
- CHECKBOX_GROUP
- RADIO_GROUP
Require images for selected and unselected state. As many built-in apps use this components I suppose that such assets already included in ROM. It's not a big problem to [download resources](https://docs.zepp.com/docs/designs/download/) and pack it into app, but in a first place - as you provided guidelines and expect consistent UI across different apps why every app should have such resources ?

This also relates to the SCROLL_LIST and it's arrow - it also have to be an image. From my point of view you could provide several built-in constants for commonly used images like forward and delete so developer would not care about sizing, coloring, masks, opacity and all that stuff.

At the end it would lead to much more consistent UIs and smaller application bundles.

**Comment by MyoungXUE** [comment]
Hello,

Thank you for your great suggestion. We appreciate your feedback and will take your request into consideration. Once again, thank you for your input.
